### ISSUE TITLE
Duplicate docstring definitions on hover while using the Python extension with another LSP

### ISSUE THREAD

**Issue by C0rn3j** [microsoft/pyright#8639]
I wish to use Python extension for the LSP for docstring hover definitions, however Pyright only has an option to disable everything EXCEPT hover.

This leads to duplicated docstring definitions, top is Pyright (oddly unformatted and lacking type defs, hence me wishing to disable it), bottom is Python with Jedi as LSP:

[Screencast_20240802_152444.webm](https://github.com/user-attachments/assets/a2e1771b-5f99-46a7-b467-c7e5071067ad)

```python
class Task:
	"""A class to represent a task.

	Attributes
	----------
	id :          The unique identifier for the task.
	... 
	"""

	def __init__(self,
		id: str,
		title: str,
		body: str|None,
		URL: str,
		priority: int,
		serviceType: str,
		serviceData: dict | None = None,
		parentId:str|None = None,
		listId:str|None = None,
		dueDate: datetime | None = None,
		timeZone: ZoneInfo | None = None
	):
```

```yaml
# VSC extensions
Pyright:  v1.1.374 
Python:  v2024.12.1

# VSC
Version: 1.92.0
Commit: b1c0a14de1414fcdaa400695b4db1c0799bc3124
Date: 2024-08-01T22:07:15.925Z
Electron: 30.3.1
ElectronBuildId: undefined
Chromium: 124.0.6367.243
Node.js: 20.15.1
V8: 12.4.254.20-electron.0
OS: Linux x64 6.9.7-arch1-1
```

**Comment by erictraut** [comment]
Also, if you have suggestions for improvements to the hover text provided by pyright, please file a feature request or bug report in the [pylance-release](https://github.com/microsoft/pylance-release) project. The pylance team is responsible for the hover text functionality. They have made a number of improvements recently for doc string formatting. If you have additional suggestions for improvements, it would be good to let them know.

**Comment by C0rn3j** [comment]
> If you want to use Jedi as your language server and pyright as a type checker only, you can disable all of its language server functionality. 

This is exactly the setup I described that leads to the aforementioned duplicated docstrings, is it not?

Or am I missing something here.

**Comment by erictraut** [comment]
Ah yes, I forgot that `pyright.disableLanguageServices` excludes hover text. I don't recall why I originally exempted hover text for this feature. I think it was because the [original request](https://github.com/microsoft/pyright/issues/122) specifically requested disabling "intellisense" (i.e. completion suggestions).

I think it's reasonable to change this feature to disable all language server features including hover text. This change will be included in the next release.

**Comment by erictraut** [comment]
This is included in pyright 1.1.375.
